src/app/shared/entry/descriptor-language.service.spec.ts

```diff
@@ -160,4 +171,41 @@ describe('Service: DescriptorLanguage', () => {
     placeholder = DescriptorLanguageService.toolDescriptorTypeEnumToDefaultDescriptorPath(<ToolDescriptor.TypeEnum>'UnrecognizedType');
     expect(placeholder).toEqual(null);
   });
+
+  it('should be able to get Tool descriptor enum from SourceFile type enum', () => {
+    let placeholder = DescriptorLanguageService.testParameterTypeEnumToToolDescriptorEnum(SourceFile.TypeEnum.SMKTESTPARAMS);
+    expect(placeholder).toEqual(ToolDescriptor.TypeEnum.SMK);
+    placeholder = DescriptorLanguageService.testParameterTypeEnumToToolDescriptorEnum(SourceFile.TypeEnum.CWLTESTJSON);
+    expect(placeholder).toEqual(ToolDescriptor.TypeEnum.CWL);
+    placeholder = DescriptorLanguageService.testParameterTypeEnumToToolDescriptorEnum(SourceFile.TypeEnum.WDLTESTJSON);
+    expect(placeholder).toEqual(ToolDescriptor.TypeEnum.WDL);
+    placeholder = DescriptorLanguageService.testParameterTypeEnumToToolDescriptorEnum(SourceFile.TypeEnum.NEXTFLOWTESTPARAMS);
+    expect(placeholder).toEqual(ToolDescriptor.TypeEnum.NFL);
+    placeholder = DescriptorLanguageService.testParameterTypeEnumToToolDescriptorEnum(SourceFile.TypeEnum.GXFORMAT2TESTFILE);
+    expect(placeholder).toEqual(ToolDescriptor.TypeEnum.GXFORMAT2);
+    placeholder = DescriptorLanguageService.testParameterTypeEnumToToolDescriptorEnum(<SourceFile.TypeEnum>'UnrecognizedType');
+    expect(placeholder).toEqual(null);
+  });
+
+  it('should be able to get descriptor path list', () => {
+    let placeholder = DescriptorLanguageService.getDescriptorLanguagesDefaultDescriptorPaths();
+    expect(placeholder).toContain('/Dockstore.cwl');
+    expect(placeholder).toContain('/Dockstore.yml'); // Galaxy
```

Comment:
think this comment kinda highlights that `Dockstore.yml` is kinda confusing as a default for galaxy, browsing through https://github.com/iwc-workflows it seems that the convention for this file is `<name of workflow>.yml` which is difficult for us to have an exact placeholder but maybe `workflow-name.yml` would be less easily to be confused with our `.dockstore.yml`

Reply:
Yeah,  if I understand correctly, this is just the placeholder text that shows up when registering a new workflow or creating a hosted workflow? I think so

Reply:
done